### ISSUE TITLE
Is this abandoned?

### ISSUE THREAD

**Issue by christianbundy** [bemyeyes/bemyeyes-android#11]
I can't find this app in the Play Store, and it looks like development has stagnated.

**Comment by epiclapser** [comment]
Hey I have an idea, why don't we just work on a copy of this code, because no ones addressing pull requests, and issues, except for a few people....

**Comment by christianbundy** [comment]
@epiclapser Has this code been released under an open-source license? Unless they add a `LICENSE` file or something, it's still under copyright and we could be taken to court for distributing a fork.

**Comment by epiclapser** [comment]
Yes I'm pretty sure it is, hold up I'll check their website again

**Comment by epiclapser** [comment]
http://www.bemyeyes.org/faq/
This clearly says it's open source.

**Comment by christianbundy** [comment]
It says "open source", but *at best* the only thing that mean is that  is that they follow the [OSD](https://opensource.org/osd-annotated). Regardless, they need to add a proper license to the project itself -- if we distribute a fork and someone asks how it's licensed, pointing them to that page isn't a sufficient answer.

**Comment by epiclapser** [comment]
https://github.com/epiclapser/bemyeyes-android

**Comment by epiclapser** [comment]
Let me email them, and maybe they'll say something about the licence....makes sense...

**Comment by aurium** [comment]
+1

**Comment by epiclapser** [comment]
Yeah they still haven't emailed back, I could try again maybe if more people email them. You guys tried anything yet?
